setup.py

```diff
@@ -102,8 +69,29 @@
     )
 )
 
-# Translation files
-mo_dirs = [x for x in glob.glob(os.path.join("languages", "*")) if isdir(x)]
+# Documentation
+docfiles = glob.glob("[!404.md]*.md") + glob.glob(os.path.join("doc", "*.md"))
```

Comment:
Thanks @mathiascode. This is better if the build environment finds the documentation. The `debian/docs` is really only for upstream packages that ship documentation in non-standard locations. I haven't tested the built package, but you're confident the documentation still ships correctly?